src/main/java/org/tikv/common/region/RegionStoreClient.java

```diff
@@ -1335,6 +1338,22 @@ private void rawDeleteRangeHelper(RawDeleteRangeResponse resp) {
     }
   }
 
+  public static boolean isMppAlive(ManagedChannel channel) {
```

Comment:
Why add this method here ? It seems to have nothing to do with `RegionStoreClient`.

Reply:
I also think it is not a good place to add this method. But I did not find a suitable place. Is there any suggestions?